### DISCUSSION TITLE
Android mini client screen rotation not working on tablet

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#38] [Android client]
Using a Samsung Galaxy S6 Lite, the SageTV Mini client display isn't auto rotating when the unit is rotated.  Yes, settings/display is set correctly.
 Anybody else having this problem?

**Comment by LoisGNS** [comment]
I have the same device, and have used Mini Client on it, as well as several other tablets & phones. In my experience, it has never auto-rotated.

**Reply by GitOutWest** [reply]
I'm using the SageTV mini client on my Samsung 10 Android 12 phone and beat it, as well as some other apps, into rotation-submission using an app (https://play.google.com/store/apps/details?id=com.pranavpandey.rotation&pli=1;   free with ads, $3 for the key to remove).

**Comment by Sagetvmaster** [comment]
I wonder how hard of an improvement this would be to add to the current Mini Client release.  Because many tablet cases/stands don’t lend themselves to being in anything but one position, it would certainly help in that regard.

**Reply by nyplayer** [reply]
I do not think this is still being supported.

**Reply by Sagetvmaster** [reply]
The mini-client?  The app that's available for Android and Amazon Firestick devices?  That's news to me if Josh has stopped support.

**Reply by eelpout** [reply]
Josh/jvl711 hasn't been heard from since last year so I would call development on it dormant. 

As for rotation issues, I recommend the Android app ["Rotation Control".](https://play.google.com/store/apps/details?id=org.crape.rotationcontrol) it may show as incompatible, but it still works for me. I use it on a Samsung tablet for apps that don't support it on their own.

The MiniClient repository is there if anyone wants to poke around too. I thought about maybe taking a look at the subtitle stuff that doesn't work because it bugs me, but I currently don't have an Android development setup, and am at the moment too lazy to set one up. :)
